strict-equality.js

```diff
@@ -0,0 +1,5 @@
+const strictEqual = (a, b) => {
```

Comment:
`a` and `b` aren't great naming. Even `value1` and `value2` is slightly better. Generally we'd only see single letter naming in a for loop.